tests/view/document/document.js

```diff
@@ -173,7 +173,7 @@ describe( 'Document', () => {
 			const domH1 = document.createElement( 'h1' );
 
 			const viewDocument = new Document();
-			viewDocument.createRoot( 'div' );
+			viewDocument.createRoot( 'DIV' );
```

Comment:
You shouldn't touch an existing test. Unless it's clearly incorrect or a new, refactored tests will check this cases as well, an existing test shouldn't be touched.